src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/CngProperty.cs

```diff
@@ -24,7 +24,18 @@ public CngProperty(string name, byte[]? value, CngPropertyOptions options)
             Name = name;
             Options = options;
             _lazyHashCode = default(int?);
-            _value = (value == null) ? null : value.CloneByteArray();
+            _value = value.CloneByteArray();
+        }
```

Comment:
The existing ctor can now be:
```C#
public CngProperty(string name, byte[]? value, CngPropertyOptions options) : this(name, (ReadOnlySpan<byte>)value, options)
{
}
```
?

Reply:
I don't think so, from the looks of it because of the null conversion from `value` to an empty span. We don't want to change the behavior of null values to start being "empty" since there is a distinct difference between null and empty.

Reply:
> there is a distinct difference between null and empty

Ah, I hadn't looked at CloneByteArray.  Ok.